README.rst

```diff
@@ -73,7 +73,7 @@ Here's the most direct way to get your work merged into the project.
 #. Push the branch up to your fork
 #. Send a pull request for your branch
 
-Please, do not create the pull requests with translation (.po) files improvements. Fix the translation on `Zanata <https://fedora.zanata.org/iteration/view/dnf-plugins-core/master>`_ instead.
+Please, do not create the pull requests with translation (.po) files improvements. Fix the translation on `Fedora Weblate <https://translate.fedoraproject.org/projects/dnf/>`_ instead.
```

Comment:
Let's fix the english a bit? :slightly_smiling_face: 
> Please do not create pull requests with translation (.po) file improvements.

(remove the comma, remove _the_, s/files/file/)

Reply:
Fixed.